docs/markdown/ios/overlaypanel.md

```diff
@@ -0,0 +1,116 @@
+   Full overlaypanel is always the full height of the viewport. They need to be dismissed to interact with underlying content.
+2. **Partial overlay panel**
+   Partial overlay panel can be any height from 50% to 90% of the viewport. They are used when content on the overlay panel relates to the layer behind, therefore a background wash is optional. They are dismissed by swiping down or tapping outside of the overlay panel area.
+3. **Action overlay panel**
+   Action overlay panels can be any height up to 50% of the viewport. They always include a background wash and need to be dismissed to interact with the underlying content. Action overlay panels can serve as an alternative to dialogs and inline menus.
+
+<ThreeCol>
+<Group>
+<ImgContainer src="https://i.pinimg.com/originals/f4/dc/73/f4dc732c95493aa96a1ca09a2bd6c66d.png" alt="a Full overlay panel that takes up the full phone screen and shows a profile picture and the ability to see ideas from a pinner, along with other actions like downloading and changing the profile picture" />
+**Full overlay panel**
+</Group>
+
+<Group>
+<ImgContainer src="https://i.pinimg.com/originals/ce/d7/9a/ced79ab112a1934d569b535de3bd6b67.png" alt="a Partial overlay panel that can be closed and/or resized. It is for Creating a board" />
+**Partial overlay panel**
+</Group>
+
+<Group>
+<ImgContainer src="https://i.pinimg.com/originals/21/c9/2c/21c92cd25b3df51249216404a64c16bd.png" alt="An Action OverlayPanel that allows for sorting and organization" />
+**Action overlay panel**
+</Group>
+</ThreeCol>
+
+### Navigation
+
+OverlayPanels can utilize several navigation actions.
+
+1. **Close**
+   To close an overlay panel, use an icon button in the top left corner with the “close” icon.
+2. **Back**
+   To return to a previous screen, use an icon button in the top left corner with an “arrow-back” icon.
+
+3. **Button**
+   An inline Button can be used for a variety of purposes, back, close, next, save, etc.
+
+4. **Grabber**
+   A Grabber is a short gray line at the top of the overlay panel that tells the user that the overlaypanel can be collapsed or expanded.
+5. **Outside click**
+   For the Partial OverlayPanel and Action OverlayPanel the user may be able to tap outside of the overlay panel to exit.
+
+### Animation
+
+By default, OverlayPanel animates in from the bottom of the screen. It animates out when the header close button is pressed, the user swipes down or the user taps outside of the overlay panel. Visit Apple’s HIG for more information on [Animation and Motion](https://developer.apple.com/design/human-interface-guidelines/foundations/motion/).
+
+## Accessibility
+
+People use Apple’s accessibility features, such as reduced transparency, VoiceOver, and increased text size to personalize how they interact with their device. Supporting these personalizations ensures that everyone has a great user experience. See Apple’s Human Interface Guidelines and API documentation about accessibility for iOS:
```

Comment:
typo: comma needed after "…and increased text size"

Reply:
UNDOING